src/public/meeting.ts

```diff
@@ -369,4 +373,13 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.speakingStateChanged', handler);
   }
+
+  export function registerRaiseHandStateChangedHandler(handler: (raiseHandState: IRaiseHandState) => void): void {
+    if (!handler) {
+      throw new Error('[registerRaiseHandStateChangedHandler] Handler cannot be null');
+    }
+    // TODO: check what context 2d avatars loads in
```

Comment:
2D avatar loads in sidePanel. It was confirmed by Ajay. Good to remove this placeholder comment.